### DISCUSSION TITLE
Stress sign convention

### DISCUSSION THREAD

**Discussion by dineshp1** [idaholab/moose#20999] [Q&A General] [answered]
Hi,
Which stress sign convention is used in the Porous Flow module: compressive stresses as positive or -ve?

Thanks,
Dinesh P

**Comment by WilkAndy** [comment, marked as answer]
It is the same convention as TensorMechanics, which is:
- a positive strain is extensional, which (usually) results from a positive (tensile) stress;
- a negative strain is compressional, which (usually) results from a negative (compressive) stress

**Reply by dineshp1** [reply]
Thank you much for the quick reply.

**Reply by jmeier** [reply]
Please allow me ask more in detail:
- Compressive porepressures (e.g. below a free water table) have a positive sign?
- Suction is represented by porepressures having a negative sign?
